app/app.go

```diff
@@ -99,6 +99,9 @@ func (a *App) InitServer() {
 			})
 		}
 		a.srv.RunJobs()
+		if a.IsLeader() {
+			model.CreateRecurringTask("Unset DND Statuses", a.UpdateDNDStatusOfUsers, time.Duration(300)*time.Second)
```

Comment:
This uses the existing recurring task and not our modified task with an offset at a specific time. Did you plan to do it later?

Reply:
Yes, I am planning to do it once other implementation details are fleshed out.